src/profile/d3Elevation.js

```diff
@@ -407,7 +407,9 @@ const exports = function(options) {
         // Configure the d3 line.
         line = d3.line()
           .x(d => x(distanceExtractor(d)))
-          .y(d => y(linesConfiguration[name].zExtractor(d)));
+          .y(d => y(linesConfiguration[name].zExtractor(d)))
+          .defined(d => d.value);
```

Comment:
It would be clearer if the condition is changed to: `.defined(d => d.value !== null)`